.github/CODEOWNERS

```diff
@@ -226,6 +226,7 @@ receiver/filestatsreceiver/                       @open-telemetry/collector-cont
 receiver/flinkmetricsreceiver/                    @open-telemetry/collector-contrib-approvers @JonathanWamsley
 receiver/fluentforwardreceiver/                   @open-telemetry/collector-contrib-approvers @dmitryax
 receiver/githubreceiver/                          @open-telemetry/collector-contrib-approvers @adrielp @andrzej-stencel @crobert-1 @TylerHelmuth
+receiver/gitlabreceiver/                          @open-telemetry/collector-contrib-approvers @adrielp @atoulme
```

Comment:
I think you and I have talked about this on the side, but after this contribution goes through and you go through getting added as an [OpenTelemetry member](https://github.com/open-telemetry/community/blob/main/guides/contributor/membership.md#requirements), it'd be great to have you added to the list of code owners for this receiver.

Reply:
Since this is my first bigger PR, I wasn't sure if that's already enough to become a member. So I thought I would update the code-owners with a subsequent PR once this one is merged and I became a member. 

If I can become a member already now, I'm happy to update the code-owners right away as I plan to own/maintain this component. What are your thoughts on this?

Reply:
Yup, updates in the next Pr sound good! And you can go ahead and open up the community issue to request to join now.